Kudu.Services/Deployment/PushDeploymentController.cs

```diff
@@ -31,6 +32,7 @@ public class PushDeploymentController : ApiController
     {
         private const string ZipDeploy = "ZipDeploy";
         private const string WarDeploy = "WarDeploy";
+        private const string OneDeploy = "OneDeploy";
```

Comment:
If this is moved to Constants.cs, you can use it instead of hardcoding "OneDeploy" in other files.